src/ucp/api/ucp.h

```diff
@@ -734,12 +736,13 @@ typedef enum {
                                                         operation, fail if the
                                                         operation cannot be
                                                         completed immediately */
-    UCP_OP_ATTR_FLAG_MULTI_SEND     = UCS_BIT(19)  /**< optimize for bandwidth of
+    UCP_OP_ATTR_FLAG_MULTI_SEND     = UCS_BIT(19),  /**< optimize for bandwidth of
                                                         multiple in-flight operations,
                                                         rather than for the latency
                                                         of a single operation.
                                                         This flag and UCP_OP_ATTR_FLAG_FAST_CMPL
                                                         are mutually exclusive. */
+    UCP_OP_ATTR_FIELD_PRIORITY      = UCS_BIT(20)   /**< priority field */
```

Comment:
pls expand the documentation

Reply:
Other field flags in this enum:
```
UCP_OP_ATTR_FIELD_REQUEST       = UCS_BIT(0),  /**< request field */
    UCP_OP_ATTR_FIELD_CALLBACK      = UCS_BIT(1),  /**< cb field */
    UCP_OP_ATTR_FIELD_USER_DATA     = UCS_BIT(2),  /**< user_data field */
    UCP_OP_ATTR_FIELD_DATATYPE      = UCS_BIT(3),  /**< datatype field */
    UCP_OP_ATTR_FIELD_FLAGS         = UCS_BIT(4),  /**< operation-specific flags */
    UCP_OP_ATTR_FIELD_REPLY_BUFFER  = UCS_BIT(5),  /**< reply_buffer field */
    UCP_OP_ATTR_FIELD_MEMORY_TYPE   = UCS_BIT(6),  /**< memory type field */
    UCP_OP_ATTR_FIELD_RECV_INFO     = UCS_BIT(7),  /**< recv_info field */
    UCP_OP_ATTR_FIELD_MEMH          = UCS_BIT(8),  /**< memory handle field */
```
    
Would `/**< message priority field */` be fine?

Reply:
Active message priority level